integration/__specs__/modules/components/cartPage/cartModal/cartModal.ts

```diff
@@ -0,0 +1,32 @@
+import { CartNameInput } from './cartNameInput/cartNameInput';
+import { CartPriceInput } from './cartPriceInput/cartPriceInput';
+import { CartQuantityInput } from './cartQuantityInput/cartQuantityInput';
+import { Component } from '@Core/component';
+
+export class CartModal extends Component {
+    protected selectors = {
+        nameInput: './/input[@name="name"]',
+        priceInput: './/input[@name="price"]',
+        quantityInput: './/input[@name="quantity"]',
+        createButton: './/button[contains(.,"Create")]',
+    };
+
+    public async getCartNameInput(): Promise<CartNameInput> {
```

Comment:
We don't have to create new components for the input (for example CartNameInput), we can just return 'Component'

![image](https://github.com/zebpaa/ta-academy_2/assets/129729966/3d9fa84a-bf22-4e85-8ddb-553842c5220f)